l10n/localizeTimestamp.js

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var moment = require('moment');
+var acceptLanguage = require('accept-language');
+
+/**
+ * This module contains localization utils for the server
+ **/
+module.exports = function (options) {
+  options = options || {}
+
+  if (! options.defaultLanguage) {
+    throw new Error('defaultLanguage is required')
+  }
+
+  var defaultLanguage = options.defaultLanguage;
+  var supportedLanguages = options.supportedLanguages;
```

Comment:
Not required because below `supportedLanguages = [defaultLanguage];`, but I should add a guard to  `supportedLanguages.length` though 👍